Gemfile

```diff
@@ -17,8 +17,11 @@ gem "mini_racer"
 gem "puma", "~> 4.3"
 gem "rollbar"
 gem "rails", "~> 6.0.1"
+gem "redis"
+gem "redis-namespace"
```

Comment:
The Redis configuration below doesn't appear to use namespacing..?

Reply:
This was a blind copy and paste. I have used it now however to separate dev and test redis databases.